README.md

```diff
@@ -55,9 +55,9 @@ Note: On the off chance that you have issues with running on the specific port o
 docker-compose file is configurable to allow for other ports.  This can be run in the following way, substituting 8080
 for whatever port you would like to assign:
 
-   ```bash
-   NGINX_PORT=8080 docker compose up -d
-   ```
+```bash
+NGINX_PORT=8080 docker compose up -d
+```
 
```

Comment:
nothing includes the credentials to use for initial login.  We need to include that.  Also, I don't think the komet file docker/komet-data/users.ini is being included in this, so we cannot login.  I also cannot verify that komet is working if I can't log in.